pkg/build/controller/strategy/util.go

```diff
@@ -113,20 +113,20 @@ func setupDockerSocket(pod *corev1.Pod) {
 
 // mountConfigMapVolume is a helper method responsible for actual mounting configMap
 // volumes into a pod.
-func mountConfigMapVolume(pod *corev1.Pod, container *corev1.Container, configMapName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource) {
-	mountVolume(pod, container, configMapName, mountPath, volumeSuffix, policy.ConfigMap, volumeSource)
+func mountConfigMapVolume(pod *corev1.Pod, container *corev1.Container, configMapName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource, configMapMode int32) {
+	mountVolume(pod, container, configMapName, mountPath, volumeSuffix, policy.ConfigMap, volumeSource, configMapMode)
 }
 
 // mountSecretVolume is a helper method responsible for actual mounting secret
 // volumes into a pod.
-func mountSecretVolume(pod *corev1.Pod, container *corev1.Container, secretName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource) {
-	mountVolume(pod, container, secretName, mountPath, volumeSuffix, policy.Secret, volumeSource)
+func mountSecretVolume(pod *corev1.Pod, container *corev1.Container, secretName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource, secretMode int32) {
+	mountVolume(pod, container, secretName, mountPath, volumeSuffix, policy.Secret, volumeSource, secretMode)
 }
 
 // mountCSIVolume is a helper method responsible for actual mounting csi
 // volumes into a pod.
-func mountCSIVolume(pod *corev1.Pod, container *corev1.Container, volumeName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource) {
-	mountVolume(pod, container, volumeName, mountPath, volumeSuffix, policy.CSI, volumeSource)
+func mountCSIVolume(pod *corev1.Pod, container *corev1.Container, volumeName, mountPath, volumeSuffix string, volumeSource *corev1.VolumeSource, csiVolumeMode int32) {
+	mountVolume(pod, container, volumeName, mountPath, volumeSuffix, policy.CSI, volumeSource, csiVolumeMode)
```

Comment:
CSI volumes do not have an associated mode, pass 0 here.

Reply:
Ah, good to know, thanks!